packages/block-editor/src/components/global-styles/test/utils.js

```diff
@@ -52,6 +55,41 @@ describe( 'editor utils', () => {
 				},
 			},
 		},
+		styles: {
+			background: {
+				backgroundImage: {
+					url: 'file:./assets/image.jpg',
+				},
+				backgroundAttachment: 'fixed',
+				backgroundPosition: 'top left',
+			},
+			blocks: {
+				'core/group': {
+					background: {
+						backgroundImage: {
+							ref: 'styles.background.backgroundImage',
+						},
+					},
+					dimensions: {
+						minHeight: '100px',
+					},
+				},
+			},
+		},
+		_links: {
+			'wp:theme-file': [
+				{
+					name: 'file:./assets/image.jpg',
+					href: 'https://wordpress.org/assets/image.jpg',
+					target: 'styles.background.backgroundImage.url',
+				},
+				{
+					name: 'file:./assets/other/image.jpg',
+					href: 'https://wordpress.org/assets/other/image.jpg',
+					target: "styles.blocks.['core/group].background.backgroundImage.url",
```

Comment:
Tiniest of tiny nits: it looks like there's a missing closing `'` in this string. Does that matter? I assume not since we're just in a test here and `target` isn't being looked at anywhere.

```suggestion
					target: "styles.blocks.['core/group'].background.backgroundImage.url",
```